articles/multifactor-authentication/custom.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Configuring Custom Multi-factor Authentication
+url: /multifactor-authentication/custom
+description: Examples for configuring custom MFA implementations.
+topics:
+  - mfa
+  - custom-mfa
+contentType:
+  - how-to
+  - concept
+useCase:
+  - customize-mfa
+---
+# Configuring Custom MFA
+
+You may configure a [rule](/rules) from your [Dashboard > Rules](${manage_url}/#/rules) for custom MFA processes, which allow you to define the conditions that will trigger additional authentication challenges. Rules can be used to force MFA for users of certain applications, or for users with particular user metadata or IP ranges, among other triggers.
+
+::: note
+The MFA settings defined in rules will always take precedence over the toggles in the MFA section of the Dashboard.
+:::
+
+## The provider setting
+
+The `provider` setting now defaults to `any`. This allows the toggles on the [Dashboard > MFA](${manage_url}/#/mfa) to control which providers are enabled, and the user to select from any of the enabled providers. 
```

Comment:
We don't default the `provider` value. If you don't enable 'Require MFA for all applications', you will need to set provider to `any` explicitly. Maybe this section could be like 'Requiring MFA through rules: to require MFA set the provider setting to 'any'. This ....'?

@aaguiarz wdyt?

Reply:
Reworded this section to hopefully make it clearer, let me know